manifests/params.pp

```diff
@@ -25,7 +26,7 @@
       if $::operatingsystemrelease =~ /^6\.(\d+)/ {
         fail("The ${module_name} module is not supported on Debian Squeeze.")
       }
-      elsif $::operatingsystemrelease =~ /^[78]\.(\d+)/ {
+      elsif $::operatingsystemrelease =~ /^[789]\.(\d+)/ {
```

Comment:
Wouldn't ```versioncmp($::operatingsystemmajrelease, '7') >= 0``` be better and more future proof?

Reply:
I think that would incorrectly catch the Ubuntu versions as well wouldn't it?

Reply:
This will likely break for the next version of Debian (10) as it will start to conflict with Ubuntu 10.x, logic should be broken off separately for Debian and Ubuntu.

Reply:
You're right. Early morning code reviews aren't the best idea :P